### PR TITLE
Angler AI - Add saveOrder preset + bugfixes

### PR DESCRIPTION
**Pull request by victor-ru**
Several changes were made:
* Fix the URL of the customers endpoint
* Add a new preset for `saveOrder` action
* Add user email and phone to the checkout object in the transform function
* Add billing and shipping address to the checkout object
* Send products to the line items endpoint in `saveOrder` 

## Testing

_Tested manually using the actions tester._

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/angler-ai/presets.ts [hunk 1]
```diff
@@ -156,5 +157,12 @@ export const presets: DestinationDefinition['presets'] = [
       eventName: 'form_submitted'
     },
     type: 'automatic'
+  },
+  {
+    name: 'Save Order',
+    subscribe: 'event = "Order Completed"',
```
Comment (joe-ayoub-segment) [inline packages/destination-actions/src/destinations/angler-ai/presets.ts:163]: hi @victor-ru . 
This preset subscribes to the same event name as the "Save Event - Checkout Completed" preset. 
i.e if a customer sends in a Segment event named "Order Completed" then your platform will receive both a "Save Event - Checkout Completed" and a "Save Order" event. Is this what you want?

**Comment by joe-ayoub-segment** [comment]
hi @victor-ru thanks for raising the PR. Just one query, otherwise looks good.

File: packages/destination-actions/src/destinations/angler-ai/presets.ts (hunk 1 above)
Comment (victor-ru) [reply packages/destination-actions/src/destinations/angler-ai/presets.ts:163]: hi @joe-ayoub-segment,
yes

**Comment by victor-ru** [comment]
thanks for looking, @joe-ayoub-segment
replied to your comment, please let me know if there is anything else we need to do to get this deployed

**Review by joe-ayoub-segment — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
hi @victor-ru this PR has been deployed. Please confirm that the change works as expected.

**Comment by victor-ru** [comment]
hi @joe-ayoub-segment,
yes, we can see the new preset and fields
thank you

**Pull request merged**